test/extended/s2i_ruby_test.go

```diff
@@ -0,0 +1,81 @@
+// +build s2i
+
+package extended
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	kapi "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = Describe("OpenShift Ruby S2I image", func() {
+	defer GinkgoRecover()
+	var (
+		oc       = exutil.NewCLI("mysql-create", kubeConfigPath())
+		versions = map[string]string{
+			"20-centos7": "ruby 2.0.0",
+			//"20-rhel7":   "ruby 2.0.0",
+			//"22-centos7": "ruby 2.2.2",
```

Comment:
Just a note for the future @mfojtik, this needs to be enabled once the 22 image is available on Hub.